### PR TITLE
add snippetIdToString

### PR DESCRIPTION
**Pull request by ryoppippi**
Related to: https://github.com/uga-rosa/cmp-denippet/pull/2

Create method called `snippetIdToString` to get body from snippet id

### CODE REVIEW

File: denops/denippet/main.ts [hunk 1]
```diff
@@ -228,6 +228,24 @@ export function main(denops: Denops): void {
       return parsed;
     },
 
+    async snippetIdToString(idU: unknown): Promise<string> {
+      const id = u.ensure(idU, is.String);
+      const parsed = lsputil.parseSnippet(id);
```
Comment (uga-rosa) [inline denops/denippet/main.ts]: A snippet ID is UUID.
https://github.com/uga-rosa/denippet.vim/blob/c0a74b0f84069bbd73f4676da72e3920d9560f39/denops/denippet/loader.ts#L100-L102

It should be the body of the snippet that is passed to `parseSnippet()`.

**Review by uga-rosa — CHANGES_REQUESTED** [review]

File: denops/denippet/main.ts (hunk 1 above)
Comment (uga-rosa) [reply denops/denippet/main.ts]:
```typescript
    async snippetIdToString(idU: unknown): Promise<string> {
      const id = u.ensure(idU, is.String);
      const { body, matched } = await searchSnippet(loader, id);
      if (body == null) {
        throw new Error(`Unknown snippet id: ${id}`);
      }
      const bodyStr = is.String(body) ? body : await body(denops, matched);
      const parsedBody = lsputil.parseSnippet(bodyStr);
      if (parsedBody === "") {
        throw new Error(`Failed parsing: ${bodyStr}`);
      }
      return parsedBody;
    },
```

**Comment by uga-rosa** [comment]
`body` and `str` are different concepts.
`body` is `console.log($0)`, str is `console.log()`, etc. `str` is for use in documentation, etc. As far as cmp-denippet is concerned, what you want to add seems to be an `id->body` function (It is body that passes to anonymous). In that case, you should name the function `snippetIdToBody` and remove the `lsputil.parseSnippet` part.

**Comment by ryoppippi** [comment]
Thanks!
Completed

**Review by uga-rosa — APPROVED** [review]

**Pull request merged**